foursight_core/deploy.py

```diff
@@ -150,15 +150,20 @@ def build_config(cls, stage, identity=None, stack_name=None,
         print(''.join(['Writing: ', filename]))
         with open(filename, 'w') as config_file:
             config_file.write(json.dumps(cls.CONFIG_BASE))
+        print(''.join(['Done writing: ', filename]))
```

Comment:
If it's not a lot of trouble, please import `PRINT` from `dcicutils.misc_utils` and use that in this output generally, not just because of testability (which you might or might not get to on this round) but so that we can ultimately put in some safeguard checks for stray `print` statements inserted for debugging.